emojione/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for emojione
+// Project: https://github.com/Ranks/emojione
+// Definitions by: Danilo Bargen <https://github.com/dbrgn/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare var emojione: {
```

Comment:
It looks like this is available as an npm module in addition to as a global.
Assuming it's not really meant to be mutated and you were just doing that for tests, you could do:

```ts
export as namespace emojione;

export const sprites: boolean;
...
export function unicodeToImage(str: string): string;
```

That way it can be imported with e.g. `import { unicodeToImage } from "emojione"` or used globally with e.g. `emojione.unicodeToImage`.